core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -958,21 +936,13 @@ func (o *evmTxStore) SaveFetchedReceipts(ctx context.Context, r []*evmtypes.Rece
 			transaction_index = EXCLUDED.transaction_index,
 			receipt = EXCLUDED.receipt
 		RETURNING evm.receipts.tx_hash, evm.receipts.block_number
-	),
-	updated_eth_tx_attempts AS (
-		UPDATE evm.tx_attempts
-		SET
-			state = 'broadcast',
-			broadcast_before_block_num = COALESCE(evm.tx_attempts.broadcast_before_block_num, inserted_receipts.block_number)
-		FROM inserted_receipts
-		WHERE inserted_receipts.tx_hash = evm.tx_attempts.hash
-		RETURNING evm.tx_attempts.eth_tx_id
 	)
-	UPDATE evm.txes
-	SET state = ?, error = ?
-	FROM updated_eth_tx_attempts
-	WHERE updated_eth_tx_attempts.eth_tx_id = evm.txes.id
-	AND evm_chain_id = ?
+	UPDATE evm.tx_attempts
+	SET
+		state = 'broadcast',
+		broadcast_before_block_num = COALESCE(evm.tx_attempts.broadcast_before_block_num, inserted_receipts.block_number)
+	FROM inserted_receipts
+	WHERE inserted_receipts.tx_hash = evm.tx_attempts.hash
```

Comment:
did u miss the evm_chain_id ?

Reply:
Oh ya good point. We don't need it for this update since tx_hash is unique but I can remove chainID from this method